src/MusicalScore/Graphical/VexFlow/VexFlowMusicSheetCalculator.ts

```diff
@@ -118,7 +118,7 @@ export class VexFlowMusicSheetCalculator extends MusicSheetCalculator {
             }
         }
         if (voices.length === 0) {
-            log.warn("Found a measure with no voices... Continuing anyway.", mvoices);
```

Comment:
log.warn gets printed to console for the end user (standard loglevel setting), and i don't think that's a good idea in this place, e.g. for an empty score.

Reply:
Yep you're righto